cacti/datadog_checks/cacti/__init__.py

```diff
@@ -2,6 +2,6 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 from .__about__ import __version__
-from .cacti import Cacti
+from .cacti import CactiCheck
```

Comment:
Looks like we have less than half our checks using this convention. Is it a direction we're taking?

Reply:
I think the ddev template generates them that way so yes?